taxi/views.py

```diff
@@ -1,3 +1,17 @@
+from django.http import HttpResponse
 from django.shortcuts import render
 
-# Create your views here.
+from taxi.models import Driver, Manufacturer, Car
```

Comment:
Consider adding a newline between the import statements to improve readability and adhere to PEP 8 guidelines.